state/modelmigration.go

```diff
@@ -849,10 +849,10 @@ func (st *State) LatestMigration() (ModelMigration, error) {
 	// away from a model and then migrated back.
 	if phase == migration.DONE {
 		model, err := st.Model()
-		if err != nil {
+		if err != nil && !errors.Is(err, errors.NotFound) {
 			return nil, errors.Trace(err)
 		}
-		if model.MigrationMode() == MigrationModeNone {
+		if model != nil && model.MigrationMode() == MigrationModeNone {
 			return nil, errors.NotFoundf("migration")
 		}
```

Comment:
I think we should handle the errors.NotFound as a separate if/else statement and then add a comment. I think we probably just want to `return nil, nil`. I'd also like a log message that states that the model is gone, can be set to debug.

Reply:
Done, let me know what you think of the comment and debug message